### DISCUSSION TITLE
Viusal Studio 2022 无法使用热重置

### DISCUSSION THREAD

**Discussion by Gclove2000** [dotnetcore/BootstrapBlazor#1858] [Q&A] [answered]
Viusal Studio 使用 BootStrap Blazor 模板创建的项目使用热重置之后无反应，只能重新运行整个程序代码更改才会生效。

**Comment by ArgoZhang** [comment]
@Gclove2000 热更新是不是去微软的仓库提交 Issue 更合适一些？

**Reply by Gclove2000** [reply]
哦，抱歉，我去修改一下提交的地方

**Comment by ArgoZhang** [comment, marked as answer]
@Gclove2000 我这边也发现一些规律。项目中 razor 文件多大概率热更新不好用

**Comment by Gclove2000** [comment]
我解决了，用了一个很神奇的办法。我等会发一个CSDN博客来描述我如何解决的。

**Comment by ArgoZhang** [comment]
@Gclove2000 太棒了啊。快快学习一下。我这边好多大工程都无法使用热更新功能。总是报错

**Comment by Gclove2000** [comment]
[Blaozr 热重置问题解决方案](https://blog.csdn.net/qq_44695769/article/details/132059327?spm=1001.2014.3001.5501)
我不知道我这样设置到底能否彻底解决，只是我这样设置是可以的。就是在Release模式下把"如果无法自动应用更新始终重新生成"勾选上了，再改回Debug模式，就可以热重置了。我也不清楚为啥。
![在这里插入图片描述](https://img-blog.csdnimg.cn/dace3bf064124aadbbf9ca5c7297407b.png)
![在这里插入图片描述](https://img-blog.csdnimg.cn/5e78eb7096fb4e65b6f9f362bd5f08ba.png)
![在这里插入图片描述](https://img-blog.csdnimg.cn/29c5b5973f7e4a069f0bfb0234a8f1dd.gif)

如果不能解决说明我这个方法也不行。我也不确定到底是否彻底解决热重置问题

**Reply by ArgoZhang** [reply]
666

**Reply by Gclove2000** [reply]
你用了成功解决了吗？

**Reply by ArgoZhang** [reply]
我这一直都可以，但是 BB 源码工程不可以，原因是超时
